applications/source/cdc-debezium-source2/src/test/docker/mongodb/import-data.sh

```diff
@@ -0,0 +1,5 @@
+#wait for the SQL Server to come up
+sleep 10s
+
+#populate the db
+/usr/local/bin/init-inventory.sh
```

Comment:
Doesn't look like this file is in use.

Reply:
It is used in the README. 
Unlike the mysql or postgres, last time i tried I could not automate the mongodb testing as it requires manual interventions. 
This might have changed but i have very low priority in testing mongo stuff.